### PR TITLE
feat: investigate configuration webpack

### PR DESCRIPTION
**Pull request by Nemo-Git-Hub**
# Investigate configuration webpack
[CN (Card_Number) - 27](https://trello.com/c/04mL09D4)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Website 2</title>
+    <link rel="stylesheet" type="text/css" href="../src/scss/main.scss" />
```
Comment (asd-ez) [inline src/index.html]:
```suggestion
    <link rel="stylesheet" type="text/css" href="./scss/main.scss" />
```

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  </head>
+  <body>
+    <div class="test">
+      <p>Hello, world!<br />And hello peaple!</p>
+    </div>
+
+    <script src="../src/js/index.js"></script>
```
Comment (asd-ez) [inline src/index.html]:
```suggestion
    <script src="./js/index.js"></script>
```

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+const path = require("path");
+const HTMLWebpackPlugin = require("html-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+// const CopyWebpackPlugin = require("copy-webpack-plugin");
```
Comment (asd-ez) [inline webpack.config.js]: Write comments or get rid

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    //   patterns: [
+    //     {
+    //       from: path.resolve(__dirname, "src/assets"),
+    //       to: path.resolve(__dirname, "dist"),
+    //     },
+    //     // { from: "other", to: "public" },
+    //   ],
+    // }),
+  ],
+  module: {
```
Comment (asd-ez) [inline webpack.config.js:44]: Add image loader

**Review by asd-ez — CHANGES_REQUESTED** [review]

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              hmr: isDev,
+            },
+          },
+          "css-loader",
+        ],
+      },
```
Comment (asd-ez) [inline webpack.config.js]: Do you have `.css` files?

**Review by asd-ez — CHANGES_REQUESTED** [review]
Get rid from .css loader

File: webpack.config.js (hunk 5 above)
Comment (Nemo-Git-Hub) [reply webpack.config.js]: sass-loader Loads a Sass/SCSS file and compiles it to CSS.
After that they are processed by the another loaders and plugins.

Comment (asd-ez) [reply webpack.config.js]: I can't see the `sass-loader` here. What did you mean?

**Review by asd-ez — APPROVED** [review]

**Pull request merged**